content/bd/0003-er-model.md

```diff
@@ -72,16 +72,16 @@ Podemos representar esta associação com a seguinte notação:
 As associações têm normalmente um nome, único em todo o diagrama, que normalmente
 é um verbo.
 
-As associações são adirecionais, mas podemos utilizar os símbolos `>` e `<` para
+Para além disso, não têm direção, mas podemos utilizar os símbolos `>` e `<` para
 desambiguar alguns casos.
```

Comment:
I don't think the word "adirecional" exists in european portuguese. Might be wrong, though.

Reply:
Adirectional doesn't seem to be a word in any language, English or Portuguese, but it's explicitly used within the course and its intended meaning is clear so I'm in favor of keeping it.

Up to @diogotcorreia.

Reply:
I think it's clear as it is. Even thought it might be used in the course and it's pretty easy to understand what is means, we don't need to use non-existent words